examples/sandbox/src/App.js

```diff
@@ -73,6 +73,7 @@ const config = {
       nps_link: { raw: {} },
       states: { raw: {} },
       date_established: { raw: {} },
+    //  image_url:{raw:{}},
```

Comment:
I just added image_url to this data set so that this should work.

Reply:
Also, make sure the formatting of this line matches the formatting of the other lines, including spaces.